hangman/lib/hangman.ex

```diff
@@ -1,9 +1,25 @@
 defmodule Hangman do
-  alias Hangman.Game, as: Game
+
+  use GenServer
+  
+  def start_link(default \\ []) do
+    GenServer.start_link(Hangman.Server, default, name: __MODULE__)
+  end
+
+  def new_game() do
+    GenServer.call(__MODULE__, {:new_game})
```

Comment:
This calls the server with :new_game, but the server looks for :new.

As a result, the app will not run.

I'm surprised that you didn't spend some more time trying to get at least one of the apps working.

# -20